src/Common/Dust.cpp

```diff
@@ -933,7 +933,7 @@ int DustInterpolant<ndim, ParticleType, StoppingTime, Kernel>::DoInterpolate
     // list error in order to generate a larger neighbour list (not properly implemented yet).
     if (h > hmax) return 0;
 
-  } while (h > h_lower_bound && h < h_upper_bound);
```

Comment:
This same inequality is also in the main grad-h (and maybe meshless?) h-finding also.  Could it cause a problem there also, or was there a reason it was bad news here?  I'll have a look at it in any case.